packages/design-system/src/Button/Button.constants.ts

```diff
@@ -0,0 +1,15 @@
+import { CLASS_NAME_PREFIX } from "__config__/constants";
+
+// classnames
+export const ButtonClassName = CLASS_NAME_PREFIX + "-button";
+
+export const ButtonLoadingClassName = ButtonClassName + "__loading";
```

Comment:
quick question: Will you reuse these classes? I am thinking of a use case where you will need to resuse these classes so that storing them in constants makes sense

Reply:
I'm using this in some styling as well.